src/action.js

```diff
@@ -166,26 +188,31 @@ function serve(programInfo) {
           console.log(`...Built ${directoryName}`);
         } else if (filePath.endsWith('meta.json')) {
           // refetch meta and then build
-          const meta = getContentMeta(directoryName, programInfo.abellConfigs.contentPath);
+          const meta = getContentMeta(
+            directoryName,
+            programInfo.abellConfigs.contentPath
+          );
           programInfo.vars.$contentObj[directoryName] = meta;
-          const indexToChange = programInfo.vars.$contentArray
-            .findIndex(content => content.$slug == directoryName);
-          programInfo.vars.$contentArray[indexToChange] = meta; 
+          const indexToChange = programInfo.vars.$contentArray.findIndex(
+            (content) => content.$slug == directoryName
+          );
```

Comment:
Can you fix the indents here? Also is there any way to roll back the prettier changes? It sometimes gets confusing of what file was changed in the PR when prettier adds these extra lines

Reply:
prettier indents have been reversed where they were breaking older functionality